journal/modernized/14-gods-great-love.adoc

```diff
@@ -1,14 +1,23 @@
 == God`'s Great Love Unto Mankind, / Through Jesus Christ, Our Lord.
 
-"`And there came a voice out of the cloud, saying this (i. e. Christ) is my beloved Son,
-hear him.`" Luke 9:35.
+[quote.scripture, , Luke 9:35]
+____
+And there came a voice out of the cloud, saying this (i. e. Christ) is my beloved Son,
+hear him.
+____
 
-"`If you love me, keep my commandments.`" John 14:16.
+[quote.scripture, , John 14:16]
+____
+If you love me, keep my commandments.
+____
 
-"`For God so loved the world, that he gave his only begotten Son,
-that whosoever believed in him might not perish, but have everlasting life.`" John 3:16.
+[quote.scripture, , John 3:16]
+____
+For God so loved the world, that he gave his only begotten Son,
+that whosoever believed in him might not perish, but have everlasting life.
+____
 
-=== PREFACE.
```

Comment:
I think I prefer to have prefaces centered. I know that the default heading 3 is left justified, but when it's a title of a section, I think it almost always looks better to put the [.centered] tag over it. 👍🏻

Reply:
No problem, I agree.